src/react-components/src/components/Image/Image.tsx

```diff
@@ -34,5 +38,23 @@ export default function Image(props: ImageProps) {
         alt: isDecorative ? "" : altText,
     };
 
-    return <img {...imageProps} />;
+    return (
+        <>
+            <img {...imageProps} />
+            {(imageCaption || imageCredit) && (
+                <div className={bem(image_base_class, [], "annotation")}>
+                    {imageCaption && (
+                        <p className={bem(image_base_class, [], "caption")}>
+                            {imageCaption}
+                        </p>
+                    )}
+                    {imageCredit && (
+                        <p className={bem(image_base_class, [], "heading")}>
```

Comment:
I would use `credit` as a blockName over `heading` for this.

Reply:
I renamed this blockName to `credit`.